docs/package-overview.md

```diff
@@ -0,0 +1,49 @@
+# Overview of PHP-HTTP packages
+
+All packages are installed via [Composer](http://getcomposer.org/).
+
+
+## httplug
+
+**Interfaces for HTTP Client**
+
+- Namespace: `Http\Client`
+- Repository: https://github.com/php-http/httplug
+- Documentation: (Httplug)[httplug.md]
+
+
+## message-factory
+
+**Abstraction to create PSR-7 requests without binding to a specific implementation**
+ 
+- Namespace: `Http\Message`
+- Repository: https://github.com/php-http/message-factory
+- Documentation: [Message Factory](message-factory.md) 
+
+
+## *-client, *-adapter
+
+**Each client implementation is separated into its own package. This allows for clean requirement specification.**
+
+- Namespace: `Http\Client`
```

Comment:
OK so most likely you wouldn't have multiple installed at once so it 
shouldn't be an issue.

Not sure if using a namespace like "Http" is a great idea though, given 
this is just a common name and not really vendor specific, but up to you :)